server/liveblog/themes/themes_assets/angular/liveblog-embed/resources.service.js

```diff
@@ -40,20 +40,44 @@
 
     Posts.$inject = ['$resource', 'config', 'users'];
     function Posts($resource, config, users) {
+        function _completeUser(obj) {
+            if (obj.name) {
+                obj.original_creator = {display_name: obj.name};
+            } else if(obj.original_creator !== "" && obj.original_creator !== 'None'){
```

Comment:
Why `None`? this should be `null` or `underfined`, but not `None` right ?
And no double quote in `obj.original_creator !== ""` please